index.html

```diff
@@ -564,75 +568,74 @@ <h6>Other content you may like:</h6>
+                        fill="currentcolor"></path>
+                </svg>
+            </div>
+            <div id="stripe-logo" class="bf-maincta-logo"><svg width="68" height="30" viewBox="0 0 68 30" fill="none"
+                    xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd"
+                        d="M67.561 15.424c0-4.804-2.327-8.595-6.775-8.595-4.467 0-7.169 3.791-7.169 8.557 0 5.65 3.19 8.502 7.77 8.502 2.233 0 3.922-.507 5.198-1.22v-3.753c-1.276.638-2.74 1.032-4.598 1.032-1.82 0-3.435-.638-3.64-2.853h9.176c0-.244.038-1.22.038-1.67zm-9.27-1.783c0-2.12 1.294-3.003 2.476-3.003 1.145 0 2.365.882 2.365 3.003H58.29zM46.373 6.83c-1.839 0-3.02.862-3.678 1.462l-.244-1.163h-4.129v21.882l4.692-.994.02-5.31c.675.487 1.67 1.181 3.321 1.181 3.36 0 6.418-2.702 6.418-8.652-.019-5.442-3.115-8.407-6.4-8.407zm-1.126 12.93c-1.107 0-1.764-.395-2.214-.883l-.019-6.962c.488-.545 1.164-.92 2.233-.92 1.708 0 2.891 1.914 2.891 4.373 0 2.515-1.164 4.391-2.89 4.391zM31.865 5.721l4.711-1.013V.898l-4.71.995v3.828zm0 1.427h4.711v16.421h-4.71V7.148zm-5.046 1.388l-.3-1.388h-4.054v16.421h4.692V12.44c1.107-1.445 2.984-1.182 3.566-.976V7.148c-.601-.226-2.797-.638-3.904 1.388zm-9.385-5.461l-4.58.976-.018 15.033c0 2.777 2.084 4.823 4.86 4.823 1.54 0 2.666-.282 3.285-.62v-3.81c-.6.245-3.565 1.108-3.565-1.67v-6.662h3.565V7.148h-3.565l.018-4.073zm-12.686 8.84c0-.732.6-1.014 1.595-1.014 1.426 0 3.228.432 4.654 1.201v-4.41C9.44 7.073 7.901 6.83 6.343 6.83 2.533 6.83 0 8.82 0 12.14c0 5.18 7.131 4.354 7.131 6.587 0 .863-.75 1.145-1.801 1.145-1.558 0-3.547-.638-5.124-1.502v4.467c1.745.751 3.51 1.07 5.124 1.07 3.903 0 6.587-1.933 6.587-5.293-.019-5.592-7.169-4.598-7.169-6.699z"
+                        fill="currentcolor"></path>
+                </svg>
+            </div>
+        </div> -->
+        <h2>Join 80,000+ businesses that use Buffer to build their brand on social media</h2>
```

Comment:
Ok, I am seeing the number mod, but please be sure to commit everything separated

Reply:
yes, i do it!